.travis.yml

```diff
@@ -4,6 +4,10 @@ compiler:
   - clang
   - gcc
 
+env:
+  - BUILD_TYPE=Release
+  - BUILD_TYPE=Debug
```

Comment:
indeed, that was in this branch

https://github.com/casacore/casacore/tree/valgrind

may still be interesting to contact travis support.